src/v3/test/integration/__snapshots__/authenticator-verification-select-authenticator.test.tsx.snap

```diff
@@ -1709,14 +1709,14 @@ exports[`authenticator-verification-select-authenticator renders form 1`] = `
                         class="authenticator-description infoSection MuiBox-root emotion-8"
                       >
                         <h3
-                          class="MuiTypography-root MuiTypography-h3 authenticator-label emotion-25"
+                          class="MuiTypography-root MuiTypography-h3 authenticator-label no-translate emotion-25"
```

Comment:
I don't think this should have the `no-translate` class. 🤔 

This string has a key in all 27 languages.

Reply:
In v2 it looks like it puts the `no-translate` class on all auth buttons [here](https://github.com/okta/okta-signin-widget/blob/2e253978624081b80e01d0433a7b0245fca1eb94/src/v2/view-builder/components/AuthenticatorVerifyOptions.js#L30)

Reply:
Oh it sure does, interesting. Ty!